bigquery_etl/parse_udf.py

```diff
@@ -34,32 +34,54 @@ class RawUdf:
     def from_file(filepath):
         """Read in a RawUdf from a SQL file on disk."""
         dirpath, basename = os.path.split(filepath)
+
+        with open(filepath) as f:
+            text = f.read()
+
+        sql = sqlparse.format(text, strip_comments=True)
+        statements = [s for s in sqlparse.split(sql) if s.strip()]
```

Comment:
This may still be vulnerable to unexpected newlines or multiple spaces in the source. In publish_views, for example, we have to do the following in order to catch edge cases:

    " ".join(tokens[0].normalized.split()) == "CREATE OR REPLACE"